components/pricing_modal/starter_disclaimer_cta.tsx

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {useIntl} from 'react-intl';
+import {useDispatch} from 'react-redux';
+import styled from 'styled-components';
+
+import {openModal, closeModal} from 'actions/views/modals';
+import CloudUsageModal from 'components/cloud_usage_modal';
+
+import {ModalIdentifiers} from 'utils/constants';
+import {FileSizes} from 'utils/file_utils';
+import {t} from 'utils/i18n';
+
+const STARTER_MESSAGES_LIMIT = 10000;
+const STARTER_FILE_STORAGE_LIMIT = 10;
+const STARTER_BOARDS_VIEW_LIMIT = 5;
+const STARTER_BOARD_CARDS_LIMIT = 500;
+const STARTER_INTEGRATIONS_LIMIT = 5;
```

Comment:
If we can't rely on the current product limits, then we probably need to enable some endpoint to get a limits per product to avoid duplicating the limits definition here. The source of truth for limits is in the products configured in CWS/Stripe. We can leave out of scope for this now but we should have a follow up ticket for it